playbooks/roles/streisand-gateway/tasks/main.yml

```diff
@@ -128,6 +128,12 @@
         state=directory
         recurse=yes
 
+- name: Verify libreswan compilation has completed
```

Comment:
In that case, place this task at the bottom of the ` streisand/playbooks/roles/l2tp-ipsec/tasks/main.yml`